src/ioctls/vcpu.rs

```diff
@@ -376,6 +376,11 @@ impl VcpuFd {
     ///
     #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
     pub fn get_cpuid2(&self, num_entries: usize) -> Result<CpuId> {
+        if num_entries > KVM_MAX_CPUID_ENTRIES {
+            // Returns the same error the underlying `ioctl` would have sent.
+            return Err(errno::Error::new(ENOMEM));
```

Comment:
nit: same here, `E2BIG` is better

Reply:
Same as above. I think `ENOMEM` is correct.